sinch/domains/voice/models/applications/requests.py

```diff
@@ -0,0 +1,34 @@
+from typing import List
+from dataclasses import dataclass
+from sinch.core.models.base_model import SinchRequestBaseModel
+
+
+@dataclass
+class AssignNumbersVoiceApplicationRequest(SinchRequestBaseModel):
+    numbers: List[str]
+    application_key: str
+    capability: str
+
+
+@dataclass
+class UnassignNumbersVoiceApplicationRequest(SinchRequestBaseModel):
+    number: str
+    application_key: str
+    capability: str
```

Comment:
I suppose we can make this a literal?

Reply:
Spec file describes capability as `Enum`. I have some enums for the Voice API already coded but I'm not enforcing them (I treat them as a sort of helpers).

In cases like `domain` property I'm using Literals (it's not marked as enum in spec):
```
   domain:
          type: string
          description: can be either “pstn” for PSTN endpoint or “mxp” for data (app or web) clients.

```
So maybe for the consistency I'll just add a `Capability ` enum to the voice API `enums.py` file? Wdyt?

Reply:
yeah, please add them for the consistency and type guidance

Reply:
Capability enum added 👌